packages/shared-state-async/files/usr/libexec/rpcd/shared-state-async

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env lua
+--[[
+Shared State Async
+
+Copyright (c) 2024  Javier Jorge <jjorge@inti.gob.ar>
+Copyright (c) 2024  Instituto Nacional de Tecnolog..a Industrial
+Copyright (C) 2024  Asociacion Civil Altermundi <info@altermundi.net>
+
+This is free software, licensed under the GNU AFFERO GENERAL PUBLIC LICENSE Version 3
+]]
+--
+local utils = require('lime.utils')
+local json = require 'luci.jsonc'
+
+local ERROR_LOG_FILE_NAME = "/tmp/shared_state_rpcd_error.log"
+
+local function parseErrors()
```

Comment:
This way of handling errors doesn't seems robust to me :-/, what if a few calls are performed in parallel and some fails and some not?
shared-state-async command return the error code to the calling shell if some error happens, in that case the error should be bubbled up without more parsing

Reply:
if this prove too cumbersome to do in Lua, consider rewriting the whole ubus plugin using ucode, or ash which are supported by default by OpenWrt

Reply:
commit e891527014394e7c0b06b1554a14fb0727e49810 implements a new way of handling, what do you think?